tests/performance/hubtest/hubtest_test.go

```diff
@@ -0,0 +1,96 @@
+/*
+Copyright 2019 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package hubtest
+
+import (
+	"github.com/kubeedge/kubeedge/tests/e2e/utils"
+	. "github.com/kubeedge/kubeedge/tests/performance/common"
+	"github.com/kubeedge/kubeedge/tests/stubs/common/constants"
+	"github.com/kubeedge/kubeedge/tests/stubs/common/types"
+
+	"github.com/golang/glog"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/api/core/v1"
+)
+
+var _ = Describe("KubeEdge hub performance test", func() {
+	Context("Test different numbers of Pods on different numbers of Edge Nodes", func() {
+		// Init params
+		var podlist metav1.PodList
+		var numOfEdgeNodes int
+		var numOfPodsPerEdgeNode int
+		var podsInfo map[string]types.FakePod
+
+		BeforeEach(func() {
+			// Create Edge Nodes
+			numOfEdgeNodes = 10
+			podlist = HandleEdgeDeployment(cloudHubURL, ctx.Cfg.ApiServer2+DeploymentHandler, ctx.Cfg.ApiServer+NodeHandler,
+				ctx.Cfg.ApiServer2+ConfigmapHandler, ctx.Cfg.EdgeImageUrl, ctx.Cfg.ApiServer2+AppHandler, numOfEdgeNodes)
+		})
+
+		AfterEach(func() {
+			// Delete Pods
+			for _, p := range podsInfo {
+				DeleteFakePod(controllerHubURL, p)
```

Comment:
how do we validate the fake pod deletion successfull or not ??

Reply:
Thanks @pavan187 added it:)